docs/legacy/porting-guide.md

```diff
@@ -185,9 +185,9 @@ The debug module has been removed. Debug functionality is now built-in (and dyna
 | ------------------------------- | ---------------------------------- | ------------------------------------------------ |
 | Properties                      |
 | `props.gl`                      | `props.device`                     | Now accepts a `Promise<Device>`.                 |
-| `props.glOptions`               | `luma.createDevice(<options>)`     |
+| `props.glOptions`               | `luma.createDevice(<options>)`     | See [CreateDeviceProps](../api-reference/core/luma.md#createdeviceprops) |
 | `props.createFramebuffer`       | N/A                                | You will need to create framebuffers explicitly. |
-| `props.debug`                   | `luma.createDevice({debug: true})` | Device debug functionality is improved.          |
+| `props.debug`                   | `luma.createDevice({webgl: {debug: true}, type: 'webgl'})` | Device debug functionality is improved. |
```

Comment:
Revert if keeping debug on the top level. The type WebGL would still be required, right?

Reply:
> Revert if keeping debug on the top level. The type WebGL would still be required, right?

Fixed, thanks